docs/config/lua/keyassignment/InputSelector.md

```diff
@@ -22,6 +22,45 @@ upon the input.
   objects from the current pane and window, and `id` and `label` hold the
   corresponding fields from the selected choice. Both will be `nil` if
   the overlay is cancelled without selecting anything.
+* `fuzzy` - a boolean that defaults to `false`. If `true`, InputSelector will start
+  in its fuzzy finding mode (this is equivalent to starting the InputSelector and
+  pressing / in the default mode).
+* `alphabet` - a string of unique characters. The characters in the string are used
```

Comment:
Ah, one last thing to wrap this up, is to annotate these new fields with version information.  Something like this will work, but putting the field, description and version into a table might end up looking better.

```suggestion

{{since('nightly')}}

These additional fields are also available:

* `alphabet` - a string of unique characters. The characters in the string are used
```

The `since` macro accepts a version number or the special `nightly` string.  I use `nightly` to find and update the docs when I tag a release.

If you want to do this table style, you can use this for the version in which InputSelector was originally added, and to have it emit as an inline version number:

```
{{since('20230408-112425-69ae8472', inline=True)}}
```

Reply:
I have inlined some version information in the key table now.